common/rdm/RDMCommandSerializer.cpp

```diff
@@ -36,6 +36,36 @@ unsigned int RDMCommandSerializer::RequiredSize(
   return sizeof(RDMCommandHeader) + command.ParamDataSize() + CHECKSUM_LENGTH;
 }
 
+bool RDMCommandSerializer::Pack(const RDMCommand &command,
+                                ola::io::ByteString *output) {
+  const unsigned int packet_length = RequiredSize(command);
+  if (packet_length == 0) {
+    return false;
+  }
+
+  size_t front = output->size();
+
+  RDMCommandHeader header;
+  PopulateHeader(&header, command);
+
+  output->append(reinterpret_cast<const uint8_t*>(&header), sizeof(header));
+  output->append(command.ParamData(), command.ParamDataSize());
+
+  uint16_t checksum = START_CODE;
+  for (unsigned int i = front; i < output->size(); i++) {
+    checksum += (*output)[i];
+  }
+  checksum = command.Checksum(checksum);
+  output->push_back(checksum >> 8);
```

Comment:
Yes, I was just thinking you could drop the checksum intermediate value and have two intermediates to go into the push_back, but it may not be worth it.